flutter_application_1/lib/main.dart

```diff
@@ -11,7 +11,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Flutter Demo App',
```

Comment:
Code ini kayanya kurang tepat, alangkah baiknya, seperti ini
```suggestion
      title: 'Flutter Demo Apps',
```